### DISCUSSION TITLE
Peace TV English

### DISCUSSION THREAD

**Discussion by darrencycheung** [iptv-org/community#376] [Show and tell]
### Channel ID

PeaceTVEnglish.ae

### Stream URL

https://livestream.peacetv.tv/live/english.smil/playlist.m3u8

### Notes

Must have

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by AntiPontifex** [comment]
Stream does not work.

**Comment by freearhey** [comment]
https://livestream.peacetv.tv/live/english.smil/playlist.m3u8 (offline)
<img width="418" alt="image" src="https://user-images.githubusercontent.com/7253922/232255032-cf16ecc9-0b36-4469-8b02-095e7ba9a28f.png">
